crates/net/network/src/transactions/mod.rs

```diff
@@ -278,7 +279,7 @@ impl<Pool: TransactionPool> TransactionsManager<Pool> {
             peers: Default::default(),
             command_tx,
             command_rx: UnboundedReceiverStream::new(command_rx),
-            pending_transactions: ReceiverStream::new(pending),
+            imported_transactions: ReceiverStream::new(pending),
```

Comment:
this is pending txs
```suggestion
            pending_transactions: ReceiverStream::new(pending),
```

Reply:
docs weren't helpful, copy-paste bug.

https://github.com/paradigmxyz/reth/blob/a4c034903b7d136b237a4f8e284d9eace43ae724/crates/net/network/src/transactions/mod.rs#L226-L227

but yeah `inserted` is correct `imported` is wrong. then we should also maybe use `pending_pool_inserts` instead for `pending_pool_imports`?

Reply:
indeed, `pending_pool_insertions` is a much more helpful name, since the code that the pool import future calls in the end is `add_transaction`, same code that sends an event on the receiver stream `inserted_transactions` (`pending_transactions`).

https://github.com/paradigmxyz/reth/blob/a4c034903b7d136b237a4f8e284d9eace43ae724/crates/transaction-pool/src/pool/mod.rs#L440-L443

the txns inserted into the pool are pending in the context of transactions pool, but it's worsening readability to refer to them as `pending` in the context of the tx manager since we already have `pending_pool_insertions` (`pending_pool_imports`). when inserted into the pool, tx manager is done, the txns are no longer pending anything in context of the tx manager. at the point the txns leave the tx manager, they become pending in the context of the transactions pool.

Reply:
will open a separate issue for the docs bug and include naming changes